display_blocks.js

```diff
@@ -34,6 +34,11 @@ dictInstPV = {
 var showPrivate = true
 var privateRunInfo = ["TITLE", "_USERNAME"]
 
+function getURLParameter(name) {
```

Comment:
Replace the hardcoded instrument title (line 68) with the actual instrument title using this method